src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3688,4 +3688,7 @@
   <data name="NotSupported_MethodBodyReplacement" xml:space="preserve">
     <value>Method body replacement not supported in this runtime.</value>
   </data>
+  <data name="NotSupported_COM" xml:space="preserve">
+    <value>Built-in COM is not supported.</value>
```

Comment:
We should probably mention that it was turned off by a feature switch. It might not be supported, but it also may be just disabled.